README.md

```diff
@@ -1 +1,22 @@
-# ArtCourses-NAN
\ No newline at end of file
+## ArtCourses-NAN ##
+
+team mates :   
+  * @AhlamKadour  
+  * @NaremanFero  
+  * @Noorb7188
+
+we are planning of building an app about Art courses and in the app we are enabling the user to choose a course to participate in.  
+
+steps :
+ - we will build our database using `build.sql`.
```

Comment:
we don't use the name of the file to build the database you should write the strategy that we will use in building it : `Postgres  SQL`